src/components/RequestGroupComposition/FormPanel.vue

```diff
@@ -0,0 +1,126 @@
+                  <i
+                    v-show="hasError"
+                    v-b-tooltip="tooltipConfig"
+                    class="fas fa-exclamation-triangle fa-lg text-danger align-middle"
+                    title="Errors in form"
+                  />
+                  <i
+                    v-show="!hasError"
+                    v-b-tooltip="tooltipConfig"
+                    class="fas fa-check fa-lg text-success align-middle"
+                    title="Section is complete"
+                  />
+                </b-col>
+                <b-col class="text-center">
+                  <h6>
+                    {{ title }} <span v-if="index > 0">#{{ index + 1 }}</span>
+                  </h6>
+                </b-col>
+                <b-col class="text-right">
+                  <b-button
+                    v-b-toggle.collapse-1
+                    v-b-tooltip="tooltipConfig"
+                    size="sm"
+                    variant="primary"
+                    :title="show ? 'Minimize' : 'Maximize'"
+                    @click="clickShow"
+                  >
+                    <i class="far" :class="show ? 'fa-window-minimize' : 'fa-window-maximize'" />
+                  </b-button>
+                  <b-button v-show="cancopy" v-b-tooltip="tooltipConfig" size="sm" class="mx-1" variant="success" title="Copy" @click="copy">
+                    <i class="fa fa-copy fa-fw" />
+                  </b-button>
+                  <b-button v-show="canremove" v-b-tooltip="tooltipConfig" variant="danger" title="Remove" size="sm" @click="remove">
+                    <i class="fa fa-trash fa-fw" />
+                  </b-button>
+                </b-col>
+              </b-form-row>
+            </b-container>
+          </b-card-header>
+          <b-card-body class="p-3">
+            <slot :show="show" />
+          </b-card-body>
+        </b-card>
+      </b-col>
+    </b-form-row>
+  </b-container>
+</template>
+<script>
+import _ from 'lodash';
+
+import { defaultTooltipConfig } from '@/util';
+
+export default {
+  name: 'FormPanel',
+  props: {
+    id: {
+      type: String,
+      required: true
+    },
+    errors: {
+      type: Object,
+      required: true
+    },
+    show: {
+      type: Boolean,
+      required: true
+    },
+    canremove: {
+      type: Boolean,
+      required: true
+    },
+    cancopy: {
+      type: Boolean,
+      required: true
+    },
+    icon: {
+      type: String,
+      required: true
+    },
+    title: {
+      type: String,
+      required: true
+    },
+    index: {
+      type: Number,
+      default: 0
+    },
+    tooltipConfig: {
+      type: Object,
+      default: () => {
+        return defaultTooltipConfig;
+      }
+    }
+  },
+  computed: {
+    hasError: function() {
+      return !_.isEmpty(this.errors);
+    }
+  },
+  methods: {
+    remove: function() {
+      // TODO: Use modal instead
```

Comment:
The intention is to use the modal that is displayed when using the `confirmMixin` in `confirmMixins.js`, instead of the browser alert feature as currently happens. I'll take care of this TODO now.